readthedocs/templates/projects/integration_webhook_detail.html

```diff
@@ -19,15 +19,11 @@
   {% if integration.has_sync and integration.can_sync %}
     <p>
       {% blocktrans trimmed %}
-        This webhook was configured when this project was imported. If this
-        integration is not functioning correctly, try resyncing the webhook:
+        This webhook was configured when this project was imported
+          or it was manually created with correct configuration. If this
+        integration is not functioning correctly, try re-syncing the webhook:
```

Comment:
This looks good, but I'd like to see us _always_ displaying the webhook URL information, so that users can confirm that the automated webhook is the one they see in their GitHub admin. We should always show this data.